frontend/pages/model/[modelId]/release/[semver]/review.tsx

```diff
@@ -0,0 +1,160 @@
+import { ArrowBack } from '@mui/icons-material'
+import { Box, Button, Container, Divider, Grid, Paper, Stack, Tooltip, Typography } from '@mui/material'
+import { useGetModel } from 'actions/model'
+import { useGetRelease, useGetReleasesForModelId } from 'actions/release'
+import { postReviewResponse, useGetReviewRequestsForModel } from 'actions/review'
+import { useGetUiConfig } from 'actions/uiConfig'
+import Markdown from 'markdown-to-jsx'
+import { useRouter } from 'next/router'
+import prettyBytes from 'pretty-bytes'
+import { useState } from 'react'
+import Loading from 'src/common/Loading'
+import ReviewWithComment, { ResponseTypeKeys } from 'src/common/ReviewWithComment'
+import UserDisplay from 'src/common/UserDisplay'
+import MultipleErrorWrapper from 'src/errors/MultipleErrorWrapper'
+import Link from 'src/Link'
+import MessageAlert from 'src/MessageAlert'
+import CodeLine from 'src/model/registry/CodeLine'
+import Wrapper from 'src/Wrapper'
+import { mutate } from 'swr'
+import { formatDateString } from 'utils/dateUtils'
+import { getErrorMessage } from 'utils/fetcher'
+
+export default function ReleaseReview() {
+  const router = useRouter()
+  const { modelId, semver }: { modelId?: string; semver?: string } = router.query
+
+  const [errorMessage, setErrorMessage] = useState('')
+
+  const { model, isModelLoading, isModelError } = useGetModel(modelId)
+  const { release, isReleaseLoading, isReleaseError } = useGetRelease(modelId, semver)
+  const { uiConfig, isUiConfigLoading, isUiConfigError } = useGetUiConfig()
+  const { mutateReleases } = useGetReleasesForModelId(modelId)
+  const { mutateReviews } = useGetReviewRequestsForModel({
+    modelId,
+    semver: semver as string,
+  })
+
+  async function handleSubmit(decision: ResponseTypeKeys, comment: string, role: string) {
+    setErrorMessage('')
+    if (!modelId || !semver) {
+      return setErrorMessage('Could not find model ID')
+    }
+
+    const res = await postReviewResponse({
+      modelId,
+      role,
+      comment,
+      decision,
+      semver,
+    })
+
+    if (!res.ok) {
+      setErrorMessage(await getErrorMessage(res))
+    } else {
+      mutate(
```

Comment:
Is this needed if we're calling `mutateReviews` below? If it is needed, could you explain why?